src/components/Link.tsx

```diff
@@ -72,6 +76,8 @@ const Link: React.FC<LinkProps> = props => {
       event.preventDefault();
       onClick(event);
     },
+    target,
+    rel: rel ?? (opensNewTab && isExternalURL) ? "noopener noreferer" : "",
```

Comment:
Is this some TS sorcery I'm not aware of or we should pass the href to the `isExternalURL` func? 😄

Reply:
We should